src/test/resources/AllOperations.xml

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+<suite name="Test for all calculator operations">
+
+    <test name="Division test">
```

Comment:
If you want, as an improvement, instead of enumerating all classes explicitly, you can point `test > packages > package`